frameworks/ruby/rails6/run.sh

```diff
@@ -5,16 +5,16 @@ set -o pipefail
 #    displaying it to the user and failing if the server rejects a query
 
 rake db:migrate
-rake db:seed
-rake user:list
-rake 'user:get[1]'
-rake user:delete_first
-rake user:list | head
-rake user:create
-rake 'user:get[101]'
-rake 'user:set_name[101, "User Name"]'
-rake 'user:get[101]'
-rake user:create_fail || true
-rake user:list | tail
+# rake db:seed
+# rake user:list
+# rake 'user:get[1]'
+# rake user:delete_first
+# rake user:list | head
+# rake user:create
+# rake 'user:get[101]'
+# rake 'user:set_name[101, "User Name"]'
+# rake 'user:get[101]'
+# rake user:create_fail || true
+# rake user:list | tail
 
```

Comment:
Instead of commenting out the previous tests, you should be able to do `rake db:rollback STEP=10` here :)

Reply:
That was by mistake, I commented them so that while testing locally it runs faster. I should have removed them before commiting. Thankyou for catching this

Reply:
Oh gotcha.  Not a problem.  I didn't notice that it was draft when I reviewed it ;)